crypto/rand/rand_unix.c

```diff
@@ -284,102 +284,102 @@ int syscall_random(void *buf, size_t buflen)
 }
 
 #if !defined(OPENSSL_RAND_SEED_NONE) && defined(OPENSSL_RAND_SEED_DEVRANDOM)
-static const char *devrandom_paths[] = { DEVRANDOM };
-static struct {
+static const char *random_device_paths[] = { DEVRANDOM };
+static struct random_device_stat {
```

Comment:
The struct name should have been introduced in b7db1b63f22d227469a71837bec206c74a8f24b3, not in this commit.